Public/Src/App/Bxl/BuildXLApp.cs

```diff
@@ -1426,7 +1426,7 @@ private void ConfigureConsoleLogging(bool notWorker, BuildViewModel buildViewMod
                     m_noLogMask,
                     onDisabledDueToDiskWriteFailure: OnListenerDisabledDueToDiskWriteFailure,
                     maxStatusPips: m_configuration.FancyConsoleMaxStatusPips,
-                    optimizeForAzureDevOps: m_configuration.OptimizeConsoleOutputForAzureDevOps || m_configuration.OptimizeWarningOrErrorAnnotationsForAzureDevOps);
+                    optimizeForAzureDevOps: m_configuration.OptimizeConsoleOutputForAzureDevOps || m_configuration.OptimizeVsoAnnotationsForAzureDevOps);

```

Comment:
>m_configuration.OptimizeConsoleOutputForAzureDevOps || m_configuration.OptimizeVsoAnnotationsForAzureDevOps [](start = 44, length = 107)

How are these 2 flags different now?

Reply:
OptimizeWarningOrErrorAnnotationsForAzureDevOps was used to control outputting error/warning message in ConsoleEventListener. But since I rolled out, we don't need it any more. 

Instead, if OptimizeConsoleOutputForAzureDevOps  or OptimizeVsoAnnotationsForAzureDevOps is set, that mean AzureDevOpsListener is in place and ConsoleEventListener should be aware of it and don't output error and warning which are taken care by AzureDevOpsListener.